query-engine/query-engine/src/main.rs

```diff
@@ -57,7 +55,7 @@ async fn main() -> Result<(), AnyError> {
         Err(PrismaError::InvocationError(_)) => {
             set_panic_hook()?;
             let ip = opts.host.parse().expect("Host was not a valid IP address");
-            let address = SocketAddr::new(ip, opts.port);
+            let addr = SocketAddr::new(ip, opts.port);
```

Comment:
Intrstng.

Reply:
Heh, guess I'm used to [Rust's naming conventions](https://doc.rust-lang.org/std/?search=addr).